core/modules/user/user.theme.inc

```diff
@@ -177,15 +177,25 @@ function theme_user_admin_roles($variables) {
     $label = $form['roles'][$role_name]['#role']->label;
     $row = array();
     if (in_array($role_name, array(BACKDROP_ANONYMOUS_ROLE, BACKDROP_AUTHENTICATED_ROLE))) {
-      $row[] = t('@name <em>(required)</em>', array('@name' => $label));
```

Comment:
I think this change is a step backwards. We've removed something that was short, easy to understand, and consistent with the rest of core, and added something foreign and confusing. Let's put this back unless we can agree on something that's better than what we had before.

Reply:
Not married to this change, but for argument's sake, this was my rationale:

>Now, as a curious user, if I click the drop-button on the first role in the list, I see no delete action available. If I try the next role, it's the same ...so do I keep going, or assume that none of the roles can be deleted? We (experienced Drupal/Backdrop users) may know what/why, but we shouldn't be assuming that it is common knowledge that the anonymous and authenticated roles cannot be deleted, whereas the rest can. To that effect, I have changed the vague "required" note next to these special-cased roles with "system role - cannot be deleted", which I think helps this situation (the respective D7 text for this was "locked").

Reply:
Yeah, I read that comment when you posted it. I still think this new text isn't as good as the vague "required" that was there before.

Reply:
I've reverted back to "required", but have removed the name placeholder and html tags from the translatable string - no need for that to be in the face of translators 😉